symbolic-debuginfo/src/sourcebundle.rs

```diff
@@ -944,12 +946,29 @@ where
                 info.set_ty(SourceFileType::Source);
                 info.set_path(filename.clone());
 
-                self.add_file(bundle_path, source, info)?;
+                collect_il2cpp_sources(&source, &mut referenced_files);
```

Comment:
`collect_il2cpp_sources` returns `Option<()>`, but there's no error handling/propagation here, is that intentional?